dotnet/src/Functions/Functions.OpenAPI/Model/RestApiOperationResponseSchema.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Json.Schema;
+
+namespace Microsoft.SemanticKernel.Functions.OpenAPI.Model;
+
+/// <summary>
+/// The REST API operation response schema
+/// </summary>
+public sealed class RestApiOperationResponseSchema
```

Comment:
I'm curious why have `RestApiOperationResponseSchema` to hold the JsonDocument with title and type, instead of just a JsonDocument on `RestApiOperationResponse`?

Reply:
Yup, good call. I had the same question as I was reducing things. I thought perhaps it'd be nice having the top level title/type available still but yeah for now will just reduce the scope of things.

Reply:
Removed.